src/config.rs

```diff
@@ -290,6 +320,7 @@ mod tests {
         let got = Config::from_str(src).unwrap();
 
         assert_eq!(got.book, book_should_be);
+        assert_eq!(got.build, build_should_be);
```

Comment:
Nice. This is exactly what I had in mind when I mentioned testing the new config format :+1: